src/main/java/edu/cornell/kfs/fp/batch/xml/AccountingXmlDocumentEntry.java

```diff
@@ -0,0 +1,159 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang.StringUtils;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = StringUtils.EMPTY, propOrder = {
+    "index",
+    "documentTypeCode",
+    "description",
+    "explanation",
+    "organizationDocumentNumber",
+    "sourceAccountingLines",
+    "targetAccountingLines",
+    "notes",
+    "adHocRecipients",
+    "backupLinks"
+})
+@XmlRootElement(name = "Document", namespace = StringUtils.EMPTY)
+public class AccountingXmlDocumentEntry {
+
+    @XmlElement(name = "Index", namespace = StringUtils.EMPTY, required = true)
+    protected Long index;
+
+    @XmlElement(name = "DocumentType", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String documentTypeCode;
+
+    @XmlElement(name = "Description", namespace = StringUtils.EMPTY, required = true)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String description;
+
+    @XmlElement(name = "Explanation", namespace = StringUtils.EMPTY, required = false)
+    protected String explanation;
+
+    @XmlElement(name = "OrganizationDocumentNumber", namespace = StringUtils.EMPTY, required = false)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    protected String organizationDocumentNumber;
+
+    @XmlElementWrapper(name = "AccountingListFrom", namespace = StringUtils.EMPTY, required = false)
+    @XmlElement(name = "Accounting", namespace = StringUtils.EMPTY, required = false)
+    protected List<AccountingXmlDocumentAccountingLine> sourceAccountingLines;

```

Comment:
should this be fromAccounitngLines?
I would expect we will add more list named well for particular documents as we implement more types

Reply:
Ok, I'll check the naming on this and change it accordingly.

Reply:
The accounting line lists are referred to as source/target in the KFS code, even if the UI is displaying them as from/to on the DI. (Also, it seems Internal Billing documents are displaying income/expense on the UI in spite of using source/target behind-the-scenes.)

I think it makes more sense to call them source/target lines in the new objects. If you think that the XML wrapper elements should be renamed to source/target, or if you think we should keep the from/to elements and add new ones later (source/target, income/expense, etc.), please let me know.

Reply:
I think I agree that using the source target naming makes sense.  However, since we will have external customers using this, I am worried about the names in the XML for transaction lists not matching the names they see on the documents in KFS.  @db483 I'd like your take on the matter

Reply:
it occurs to me,  we should keep the tag and pojo field name the same, so if we ant to keep the pojo using source/target, we should change the XML to be consistent.  It is brand new XML, so we can change it without any repercussions.

Reply:
Good point, I'll change the XML accordingly.

Reply:
I think we need to confirm what naming convention we want